### ISSUE TITLE
Failed to build on Windows

### ISSUE THREAD

**Issue by johnd0e** [LevitatingBusinessMan/openai-cli#3]
```
   Compiling reedline v0.17.0
error[E0599]: no method named `map_err` found for struct `Vec<u8>` in the current scope
  --> C:\Users\xxx\.cargo\registry\src\github.com-1ecc6299db9ec823\reedline-0.17.0\src\painting\utils.rs:40:10
   |
40 |         .map_err(|_| ())
   |          ^^^^^^^ method not found in `Vec<u8>`
```
Also tried v0.17.0, but no success either.

**Comment by LevitatingBusinessMan** [comment]
What version of the rust toolchain are you on? It should compile using the latest stable release.

**Comment by johnd0e** [comment]
Thank you, the issue is fixed updating 1.68.2 -> 1.71.1
